apps/payments/next/app/[locale]/[offeringId]/[interval]/upgrade/[cartId]/(startLayout)/start/page.tsx

```diff
@@ -16,13 +20,31 @@ import {
   SupportedPages,
 } from '@fxa/payments/ui/server';
 import { Metadata } from 'next';
+import { config } from 'apps/payments/next/config';
 
 export const dynamic = 'force-dynamic';
 
-export const metadata: Metadata = {
-  title: 'Upgrade',
-  description: 'Enter your payment details to complete your upgrade.',
-};
+export async function generateMetadata(
+  {
+    params,
+    searchParams,
+  }: {
+    params: CheckoutParams;
+    searchParams: Record<string, string> | undefined;
+  },
+): Promise<Metadata> {
+  const acceptLanguage = headers().get('accept-language');
+  return makePageMetadata(
+    params,
+    'Upgrade',
+    'Enter your payment details to complete your upgrade.',
+    'start',
+    'upgrade',
+    acceptLanguage,
```

Comment:
can the `acceptLanguage` variable simply be removed, with us passing `headers().get('accept-language')` into the function directly?